CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ set(sailfishoffice_QML_SRCS
     qml/FileListPage.qml
     qml/Main.qml
     qml/CoverFileItem.qml
+    qml.SortTypeSelectionDialog.qml
```

Comment:
Bad character, doesn't even compile without fixing it.

Reply:
Sorry. Corrected